src/olympia/devhub/views.py

```diff
@@ -1903,21 +1903,19 @@ def onboarding_subscription(request, addon_id, addon):
         )
         return http.HttpResponseServerError()
 
-    if sub.stripe_session_id != session.id:
-        fields_to_update['stripe_session_id'] = session.id
+    if sub.stripe_session_id != session['id']:
+        fields_to_update['stripe_session_id'] = session['id']
 
     if len(fields_to_update) > 0:
         sub.update(**fields_to_update)
 
-    existing_version_pending = addon.versions.filter(
+    new_version_number = request.session.get('resigned_version_string')
```

Comment:
Shouldn't we clear the session after that? I think we should because otherwise the page will always display content like:

```
 You're done! A new version 1.5.1-signed of your add-on textmarkerpro will be published as Sponsored and should become available shortly. If you have any questions, don’t hesitate to contact us.
```

I'd expect to see the "A new version [...]" message once. It should be gone after page refresh.

Reply:
I wasn't really sure how long the session would persist for - and how long we'd want to keep the message around for - but I guess it's best to clear it to tidy it up

Reply:
https://github.com/mozilla/addons-server/pull/15993/commits/811d8a62e84e95d847add46d5f4f03693aeca0e9